JASP-Engine/JASP/R/ttestpairedsamples.R

```diff
@@ -176,16 +178,30 @@ TTestPairedSamples <- function(jaspResults, dataset = NULL, options, ...) {
     p1 <- pair[[1]]
     p2 <- pair[[2]]
     
+    errors <- .hasErrors(dataset,
+                         type = c('observations', 'variance', 'infinity'),
+                         all.target = c(p1, p2), 
+                         observations.amount  = c('< 2'))
+    
     row <- list(v1 = "", sep = "", v2 = "")
     
     ## test is a number, indicating which tests should be run
     for (test in seq_len(length(optionsList$whichTests))) {
-      
       currentTest <- optionsList$whichTests[[test]]
       
       ## don't run a test the user doesn't want
       if (!currentTest)
         next
+      if (!identical(errors, FALSE)){
+        jaspResults[["ttest"]]$addFootnote(errors$message)
+        isFirst <- (rowNo %% 2 == 1 && sum(optionsList$allTests) == 2) || (sum(optionsList$allTests) == 1)
+        row <- list(v1  = ifelse(isFirst, p1, ""),
+                    sep = ifelse(isFirst, "-", ""),
+                    v2  = ifelse(isFirst, p2, ""))
+        row[[testStat]] <- NaN
+        jaspResults[["ttest"]]$addRows(row)
+        next
+      }
```

Comment:
This adds too many rows for each error; always # statistics + 1

Reply:
This is not fixed, note the third row:
<img width="1821" alt="Screenshot 2019-11-20 at 11 22 11" src="https://user-images.githubusercontent.com/18737241/69230678-04ab1480-0b88-11ea-9824-532c1a094797.png">